.github/workflows/release.yml

```diff
@@ -0,0 +1,159 @@
+name: Release
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+
+
+jobs:
+  Linux:
+    name: 'Build Linux Release'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get install ninja-build
+          sudo apt-get install gettext
+          pip3 install conan==1.36.0
+
+      - name: Conan common config
+        run: |
+          conan profile new --detect default
+          conan profile update settings.build_type=Release default
+          conan profile update settings.compiler.libcxx=libstdc++11 default
+
+      - name: Run Conan
+        run: |
+          mkdir build && cd build
+          conan profile list
+          conan profile show default
+          conan install .. -o webready=False --build missing
+
+      - name: Build packaged release
+        run: |
+          cd build
+          cmake -GNinja -DEXIV2_TEAM_PACKAGING=ON -DBUILD_SHARED_LIBS=ON -DEXIV2_ENABLE_WEBREADY=OFF -DEXIV2_ENABLE_NLS=ON -DCMAKE_BUILD_TYPE=Release -DEXIV2_ENABLE_BMFF=ON -DEXIV2_TEAM_WARNINGS_AS_ERRORS=ON ..
```

Comment:
I am just 1 question away of approving the PR. Why are you disabling `WEBREADY` in the 3 platforms? I think we have it enabled in most of the other CI jobs.

Reply:
That's again just taken from the build log of `0.27.3`. 
For reference, here is the entire command: 
```!
cmake .. -G Unix Makefiles -DEXIV2_TEAM_PACKAGING=On -DBUILD_SHARED_LIBS=1 -DEXIV2_BUILD_UNIT_TESTS=True -DEXIV2_TEAM_USE_SANITIZERS=0 -DEXIV2_ENABLE_VIDEO=0 -DEXIV2_ENABLE_WEBREADY=False -DEXIV2_ENABLE_SSH=0 -DEXIV2_ENABLE_CURL=False -DEXIV2_ENABLE_NLS=1 -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_STANDARD=98 -DCMAKE_CXX_FLAGS=-Wno-deprecated
```

Happy to change it though, do you think we should enable it?

Reply:
After thinking about it, it probably makes more sense to disable it. I would say that most people use exiv2 locally and not to process images available in the web. Fine for me to keep it as it is 👍